source/js/Directions.js

```diff
@@ -7,55 +7,220 @@ class Directions extends HTMLElement {
+          button.classList.remove('listItemStyleShown');
+          button.classList.add('listItemStyle');
+          wrapper.removeChild(up_arrow);
+          wrapper.appendChild(down_arrow);
+        }
+      });
+      
+      list.appendChild(button);
 
       // If there are inner steps, display them as well
       if (directionList[i].itemListElement != undefined) {
         for (let j = 0; j < directionList[i].itemListElement.length; j++) {
-          const innerListItem = document.createElement("li");
-          innerListItem.textContent = `${directionList[i].itemListElement[j].text}`;
-          list.appendChild(innerListItem);
+          let y = j+1;
+          let button = document.createElement('button');
+          let wrapper = document.createElement('div');
+          wrapper.classList.add('buttonstyle');
+          let text = document.createElement('p');
+          let down_arrow = document.createElement('img');
+          down_arrow.setAttribute('src', './images/arrow-down.png');
+          let up_arrow = document.createElement('img');
+          up_arrow.setAttribute('src', './images/arrowUp.png');
+          up_arrow.classList.add('upArrow');
+          down_arrow.classList.add('downArrow');
+          let cb = document.createElement('input');
+          cb.type = 'checkbox';
+          cb.classList.add('checkbox');
+          wrapper.appendChild(cb);
+          text.textContent = `${x + '.' + y +')'+ ' ' + directionList[i].itemListElement[j].text}`;
+          text.classList.add('text');
+          wrapper.appendChild(text);
+          wrapper.appendChild(down_arrow);
+          button.appendChild(wrapper);
+          //event listener to expand the directions.
+          button.classList.add('listItemStyle');
+          button.addEventListener('click', event => {
+          if(button.classList.contains('listItemStyle')){
+            button.classList.remove('listItemStyle');
+            button.classList.add('listItemStyleShown');
+            wrapper.removeChild(down_arrow);
+            wrapper.appendChild(up_arrow);
+          }else{
+          button.classList.remove('listItemStyleShown');
+          button.classList.add('listItemStyle');
+          wrapper.removeChild(up_arrow);
+          wrapper.appendChild(down_arrow);
         }
+      });
+      
+        list.appendChild(button);
+    }
       }
     }
     wrapper.appendChild(list);
-
     this.shadowRoot.appendChild(styleElem);
     this.shadowRoot.appendChild(wrapper);
   }
+  
 }
 
-customElements.define("directions-info", Directions);
+customElements.define('directions-info', Directions);
 
-/** *******************************************************************/
-/** *                       Helper Functions:                       ***/
-/** *          Shout out to the TA's lemme just yoink these         ***/
-/** *******************************************************************/
+/*********************************************************************/
```

Comment:
Codacy has a fix for the issue: [Expected space or tab after '/**' in comment.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8460933)
```suggestion
/** *******************************************************************/
```